tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "moduleResolution": "node",
     "noEmitOnError": true,
     "noImplicitAny": true,
-    "outDir": "./dist/",
+    "outDir": "./dist/asd",
```

Comment:
Part of a test. You should never use this tsconfig, it's only there for tooling.